cmd/common/utils/parser-tableformatter.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/kinvolk/inspektor-gadget/pkg/columns"
+	"github.com/kinvolk/inspektor-gadget/pkg/columns/formatter/textcolumns"
+)
+
+const (
+	KubernetesTag       string = "kubernetes"
+	ContainerRuntimeTag string = "runtime"
+)
+
+type Option func(*GadgetParserOptions)
+
+func WithMetadataTag(metadataTag string) Option {
+	return func(opts *GadgetParserOptions) {
+		opts.metadataTag = metadataTag
+	}
+}
```

Comment:
What about if we need more than one tag? Maybe change to []string?

Reply:
I would like to keep it as simple as possible knowing that it can be easily extended and I don't expect any use-case where it could be needed.